docs-src/go/worker-sessions.md

```diff
@@ -10,7 +10,8 @@ tags:
   - workers
   - go sdk
 ssdi:
-  - This feature is currently available only in the Go SDK.
+  - This feature is only available in the Go SDK.
+  - For other SDKs, use Worker-specific Task Queues. See samples in [TypeScript](https://github.com/temporalio/samples-typescript/tree/main/worker-specific-task-queues), [Python](https://github.com/temporalio/samples-python/tree/main/worker_specific_task_queues), [.NET](https://github.com/temporalio/samples-dotnet/tree/main/src/WorkerSpecificTaskQueues), and [Java](https://github.com/temporalio/samples-java/tree/main/core/src/main/java/io/temporal/samples/fileprocessing).
```

Comment:
I think it would be helpful to have a note here about worker-specific task queues. The note could  point to the SDK comparison once that exists, but for now, I believe the samples are the only place we have that talk about it.